react/features/welcome/actionTypes.js

```diff
@@ -8,3 +10,15 @@
  * }
  */
 export const SET_SIDEBAR_VISIBLE = Symbol('SET_SIDEBAR_VISIBLE');
+
+/**
+ * Action to update the default page index of the {@code WelcomePageLists}
```

Comment:
When I asked you to adopt the name `set` rather than `update`, I really meant everywhere, not where I put the comment.